source/views/sis/course-search/index.js

```diff
@@ -67,7 +67,7 @@ class CourseSearchView extends React.PureComponent<Props, State> {
 		const query = text.toLowerCase()
 		let results = this.props.allCourses.filter(course => {
 			const instructors = course.instructors
-				? course.instructors.toString().toLowerCase()
+				? course.instructors.join().toLowerCase()
```

Comment:
Okay, humor me for a second. 

```js
(course.instructors || []).some(name => name.toLowerCase().includes(query))
```

Prevents you from accidentally searching across several prof names? ???

Reply:
lol yeah

Reply:
still relevant